.github/actions/milestone-it/entrypoint.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+set -e
+
+# 1. Read current version.
+
+version=$(jq -r '.version' package.json)
+
+IFS='.' read -ra parts <<< "$version"
+major=${parts[0]}
+minor=${parts[1]}
+
+# 2. Determine next milestone.
+
+if [ minor == '9' ]; then
+	major=$((major+1))
```

Comment:
we should probably set minor to zero here

Reply:
> we should probably set minor to zero here

Yep, good call. Fixed in dc35b0298.